internal/gw/client_publish_errors_test.go

```diff
@@ -54,15 +54,15 @@ func TestClientPublishErrors(t *testing.T) {
 		gw.nextHTTPResponse = &http.Response{
 			Status:     "418 I'm a teapot",
 			StatusCode: 418,
-			Body:       io.NopCloser(strings.NewReader("")),
+			Body:       io.NopCloser(strings.NewReader("{\"detail\":[\"oops\"]}")),
```

Comment:
This didn't fix the coverage (you can see this locally with "make htmlcov").

The code path we're trying to cover is when we can't even read the response.  For example, the network drops out while we are reading the request body.  It would be required to provide a Reader object where the Read method returns an "err" value which is not nil, and not EOF.

Reply:
Ah, I misunderstood. And thank for the htmlcov tip -- didn't realize we had that!